lib/mina-circle/version.rb

```diff
@@ -1,3 +1,3 @@
 module MinaCircle
-  VERSION = '1.4.2'
+  VERSION = '2.0.0'
```

Comment:
With this change, we should provide the additional docs to the readme.

Reply:
I suggest this at best is a minor release. I will update this & the documentation.